Plugins~/Src/MeshSyncClientBlender/python/3.0.0/unity_mesh_sync.py

```diff
@@ -0,0 +1,327 @@
+bl_info = {
```

Comment:
@sindharta Do we support blender 3.0.0? If not I can delete this file. It is not in the cmakelist so maybe it's not supported or the same as blender 3.0.1?

Reply:
For the same major-minor combined version of Blender, we "officially" support only a single patch version of that combination.

In theory, what we have in 3.0.1 should work for 3.0.0 though, although we make no guarantees.

Reply:
Ok, I've removed this file.